crates/sui-graphql-e2e-tests/tests/transaction_connection/pagination.move

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//# init --simulator --accounts A
+
+//# programmable --sender A --inputs 1 @A
```

Comment:
nit: can you differentiate the transactions some other way than their cursor? (e.g. use a different split amount in each one and display that in the transaction?) Cursors are going to be obfuscated soon.

Reply:
yes good idea